08-collab.md

```diff
@@ -97,3 +97,12 @@ Fast-forward
  1 file changed, 1 insertion(+)
  create mode 100644 pluto.txt
 ~~~
+
+> ## Review changes {.challenge}
+>
+> The Owner makes some changes in the repository without giving any information
+> to the Collaborator. How can the Collaborator find out what has changed with 
+> command line? And on GitHub? 
+> 
+> The Collaborator makes some local changes and committed them. Has the process
+> to review changes changed? 
```

Comment:
It is not really clear. And when I think about it, it is not really interesting. 
I will remove it and add a new challenge on reviewing changing on GitHub (comments in commits, ...)